prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+def main():
+    user_email = input("Email: ")
+    email_dict = {}
+
+    while not user_email == "":
+
+        if "@" not in user_email:
+            print("E-mail must contain @")
+            user_email = input("Email: ")
+        else:
+            name = (user_email.split("@")[0]).split(".")
```

Comment:
could you use a function to obtain the name from the email address provided by user?

Reply:
Done, thanks.